src/service.py

```diff
@@ -188,10 +202,11 @@ def _get_keyword_window(self, sel_word: str, words_of_sentence: List, length=5)
 
         remember: sel_word is lemmatized
         """
-        if length <= 0:
+        if length <= 0 or len(words_of_sentence) <= length:
```

Comment:
Rather than finding the length of the sentence manually, the following code will divide the sentence into 5 words based on the selected word:
    ngrams = [wordlist[i:i + 5] for i in range(len(wordlist) - 4)].
Dividing the sentence into chunks of 5 words including the selected word as central point.